geomesa-tools/src/main/assemblies/component.xml

```diff
@@ -15,8 +15,23 @@
                 <exclude>org.locationtech.geomesa:geomesa-accumulo-distributed-runtime*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-compute*</exclude>
                 <exclude>com.google.guava:guava</exclude> <!-- Guava conflicts with Hadoop 2.x which provides guava 11.0.2-->
+                <exclude>org.apache.zookeeper:zookeeper</exclude>
             </excludes>
         </dependencySet>
+        <dependencySet>
+            <outputDirectory>kafka-lib</outputDirectory>
+            <directoryMode>755</directoryMode>
+            <fileMode>0644</fileMode>
+            <useProjectArtifact>false</useProjectArtifact>
+            <useTransitiveDependencies>true</useTransitiveDependencies>
+            <useTransitiveFiltering>true</useTransitiveFiltering>
+            <includes>
+                <include>com.google.guava:guava</include>
+                <include>org.apache.kafka:kafka_2.11</include>
+                <include>org.locationtech.geomesa:geomesa-kafka-datastore</include>
+                <include>org.apache.zookeeper:zookeeper</include>
```

Comment:
so i believe for accumulo scripts we decided to have this is a separate install instead of putting things in the lib dir because we wanted to use ACCUMULO_HOME or HADOOP_DIR or whatever or ZOOKEEPER_HOME if they were set. We should try to be consistent here. We usually end up installing the tools anyway. Something to at least think about